Core/Src/main.c

```diff
@@ -55,7 +57,12 @@ void SystemClock_Config(void);
 
 /* Private user code ---------------------------------------------------------*/
 /* USER CODE BEGIN 0 */
-
+//Single potentiometer on CH0
+uint8_t txData[2];
+uint8_t rxData[2];
+uint16_t size = 2;
+uint32_t timeout = 500;
```

Comment:
`timeout` should be a const if it is not going to change.

Reply:
same with `size` and any other variable or array that is never going to change. This is mainly for readability and a 'safety' to make sure that no one changes it.

Reply:
also remember to make `const` variables in ALL CAPS